CMakeLists.txt

```diff
@@ -57,5 +57,9 @@ endif()
 
 add_subdirectory(${QuokkaCode_SOURCE_DIR}/extern/amrex ${QuokkaCode_BINARY_DIR}/amrex)
 add_subdirectory(${QuokkaCode_SOURCE_DIR}/extern/fmt ${QuokkaCode_BINARY_DIR}/fmt)
+
+#add compiler definitions again because
+#they do not carry forward from Microphysics
+add_definitions(-DNAUX_NET -DSTRANG)
```

Comment:
Is there a way to avoid hard-coding this? Can this be defined as a variable in Microphysics?

Reply:
For our purposes (ISM chemistry), it looks like we will always use these. Maybe we can come back to it later when we add more networks.

Reply:
Ok, can you add a comment explaining what these definitions do / why we need them enabled?